### PR TITLE
nightly-samba-builds: trigger ci job on PR to the samba-build branch

### PR DESCRIPTION
**Pull request by obnoxxx**
Signed-off-by: Michael Adam <obnox@samba.org>

### CODE REVIEW

File: jobs/nightly-samba-rpm-builds.yml [hunk 1]
```diff
@@ -12,30 +12,30 @@
...
     properties:
-    #- github:
-    #    # TODO: change the repo to final location
-    #    url: https://github.com/obnoxxx/samba-integration
+    - github:
+        url: https://github.com/gluster/samba-integration
     - build-discarder:
         days-to-keep: 7
-        #artifact-days-to-keep: 7
+        artifact-days-to-keep: 7
```
Comment (nixpanic) [inline jobs/nightly-samba-rpm-builds.yml:19]: This does not matter much. It refers to the artifacts that Jenkins gathers. The rsync'd repository needs to be cleaned on the CentOS side.

File: jobs/nightly-samba-rpm-builds.yml [hunk 2]
```diff
@@ -12,30 +12,30 @@
...
-    #    - anoopcps9
-    #    - spuiuk
-    #    - nixpanic
-    #    cron: H/5 * * * *
-    #    status-context: centos-ci
+    - github-pull-request:
+        admin-list:
+        - obnoxxx
+        - gd
+        - anoopcps9
```
Comment (nixpanic) [inline jobs/nightly-samba-rpm-builds.yml:27]: should be @anoopcs9

**Comment by nixpanic** [comment]
Looks good, except for the username

**Comment by anoopcs9** [comment]
> Looks good, except for the username

raised #89

File: jobs/nightly-samba-rpm-builds.yml (hunk 1 above)
Comment (obnoxxx) [reply jobs/nightly-samba-rpm-builds.yml:19]: Sure, I just noticed that I had commented that out in the initial commit.
I'll leave that hunk alone for this commit.

File: jobs/nightly-samba-rpm-builds.yml (hunk 2 above)
Comment (obnoxxx) [reply jobs/nightly-samba-rpm-builds.yml:27]: good catch, thanks!

Comment (obnoxxx) [reply jobs/nightly-samba-rpm-builds.yml]: fixed

File: jobs/nightly-samba-rpm-builds.yml (hunk 1 above)
Comment (obnoxxx) [reply jobs/nightly-samba-rpm-builds.yml]: left the artifacts line alone in the updated PR

**Pull request merged**